metadata_layer.py

```diff
@@ -1429,7 +1429,17 @@ def apply(self, edition, collection, metadata_client=None, replace=None,
         so work.simple_opds_feed refresh can be triggered.
         """
         _db = Session.object_session(edition)
-        made_core_changes = False
```

Comment:
made_core_changes is the same as work_requires_new_presentation_edition, with one exception (which I think we handle better now).